### PR TITLE
added file counts to website models

### PR DESCRIPTION
**Pull request by sherwoodf**
Added file count, image count, and file-type list (using suffix, rather than biostudies format) to datasets, to display information on website.

### CODE REVIEW

File: bia-export/bia_export/website_conversion.py [hunk 1]
```diff
@@ -127,6 +127,22 @@ def create_experimental_imaging_datasets(
             },
         }
 
+        eid_counts_map = {}
+        file_reference_directory = root_directory.joinpath(
+            f"file_references/{accession_id}/*.json"
```
Comment (kbab) [inline bia-export/bia_export/website_conversion.py:132]: Will this work on Windows platforms?

Comment (sherwoodf) [reply bia-export/bia_export/website_conversion.py:132]: apparently so according to the tests?

File: bia-export/bia_export/website_conversion.py [hunk 2]
```diff
@@ -155,6 +175,18 @@ def process_details_section(
...
+            if eid_dict["uuid"] in eid_counts_map:
+                eid_counts_map[eid_dict["uuid"]]["file_type_aggregation"] = sorted(
+                    list(eid_counts_map[eid_dict["uuid"]]["file_type_aggregation"])
+                )
+                eid_dict = eid_dict | eid_counts_map[eid_dict["uuid"]]
+            else:
+                eid_dict = eid_dict | {
+                    "file_count": 0,
+                    "image_count": 0,
+                    "file_type_aggregation": set(),
```
Comment (kbab) [inline bia-export/bia_export/website_conversion.py:187]: does this need to be `[]`

**Comment by sherwoodf** [comment]
Change the model to use a list, so it keeps order, but the code uses a set (and then converts to list) in order to get unique values.

**Review by kbab — APPROVED** [review]
LGTM

**Pull request merged**